packages/node_modules/@cerebral/inferno/index.d.ts

```diff
@@ -1,12 +1,31 @@
-import { InfernoChildren } from 'inferno';
+import { InfernoChildren, Props as InfernoProps } from 'inferno';
+import InfernoComponent from 'inferno-component';
+import { ResolveValue } from 'function-tree'
+import { ControllerClass, SignalChain } from 'cerebral';
 
-export function StateContainer(opts: any): any
+type ComponentState = {} | void;
 
-export interface ContainerInterface {
-    getChildContext(): any
-    render(): InfernoChildren
+interface InfernoComponentClass<P> {
+    new (props?: P, context?: any): InfernoComponent<P, ComponentState>;
 }
-export function Container(opts: any): any
 
-export function connect(props: any, comp: (props: any) => void): any
-export function connect(props: any, comp: any): any
+interface FunctionalInfernoComponent<P> {
```

Comment:
We could remove the `Inferno` in types since the package already indicates that we are dealing with inferno. I see that the 'inferno' package has such types but we do not have this convention here for other packages. You can use the simplest types possible, like `Props`, `StatelessComponent`, `ComponentClass`, etc. This makes these typings easier to understand and maintain.